### PR TITLE
refactor: rename to library, use sdk, update yml

### LINKED ISSUE #161: Update query client config
Following this change https://github.com/graasp/graasp-query-client/pull/174 the query client config should be updated

### LINKED ISSUE #172: Rename to library

### PR DESCRIPTION
**Pull request by pyphilia**
This PR:
- updates yml workflows
- Changes `Explorer` to `Library`
- Update yarn
- small refactor
- replace utils for sdk

close #172 
close #161

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "graasp-explore",
+  "name": "graasp-library",
```
Comment (dialexo) [inline package.json:2]: Should the repo also be renamed?

File: package.json [hunk 2]
```diff
@@ -98,5 +98,5 @@
       "pages/api/__coverage__.js"
     ]
   },
-  "packageManager": "yarn@3.2.0-rc.8"
+  "packageManager": "yarn@3.2.2"
```
Comment (dialexo) [inline package.json:101]: Other repos are on `3.2.1`, should we update or pin?

**Review by dialexo — APPROVED** [review]
LGTM 🦁

**Review by spaenleh — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (pyphilia) [reply package.json:2]: Yes I guess.

**Pull request merged**